src/cloud/project/user-admin.md

```diff
@@ -232,19 +241,33 @@ To enable 2FA on your {{site.data.var.ece}} user account:
    ![Cloud TFA recovery codes]({{ site.baseurl }}/common/images/cloud/cloud-2fa-settings-tfa-recovery-codes.png){:width="550px"}
 
    {:.bs.callout-warning}
-   Copy the recovery codes to another location or write them down to recover access if you your device or access to the authenticator application. Save the codes to your account to view and manage them from your account security settings. If you lose account access on a 2FA-enabled account and have no recovery codes, you must contact your Project administrator, or submit a Magento support ticket to reset the TFA application.
+   Copy the recovery codes to another location or write them down to use if you lose access to your device or authentication app. Save the codes to your account to view and manage them from your account security settings. If you lose account access on a 2FA-enabled account and have no recovery codes, you must contact your Project administrator, or submit a Magento support ticket to reset the TFA application.
```

Comment:
Misc edits for clarification.
```suggestion
   Copy the recovery codes to another location or write them down in case you lose access to your device or authentication app. Save the codes to your account so you can view and manage them from your account security settings. If you lose account access on a 2FA-enabled account and have no recovery codes, you must contact your project administrator, or submit a Magento support ticket to reset the TFA application.
```